### ISSUE TITLE
Host Up/Down history not visible in host event logs page

### ISSUE THREAD

**Issue by anthoc** [centreon/centreon#8801]
# BUG REPORT INFORMATION

***Versions***

For the RPM based systems

```
centreon-broker-storage-20.04.5-1.el7.centos.x86_64
centreon-plugin-Applications-Monitoring-Centreon-Poller-20200602-094050.el7.centos.noarch
centreon-plugin-Applications-Protocol-Http-20200602-094050.el7.centos.noarch
centreon-20.04.3-5.el7.centos.noarch
centreon-widget-grid-map-20.04.0-5.el7.centos.noarch
centreon-connector-perl-20.04.0-2.el7.centos.x86_64
centreon-gorgone-centreon-config-20.04.2-1.el7.centos.noarch
centreon-license-manager-common-20.04.2-1.el7.centos.noarch
centreon-perl-libs-20.04.3-5.el7.centos.noarch
centreon-broker-cbmod-20.04.5-1.el7.centos.x86_64
centreon-plugin-Applications-Protocol-Ftp-20200602-094050.el7.centos.noarch
centreon-plugin-Applications-Protocol-Ldap-20200602-094050.el7.centos.noarch
centreon-plugin-Operatingsystems-Windows-Snmp-20200602-094050.el7.centos.noarch
centreon-engine-daemon-20.04.2-1.el7.centos.x86_64
centreon-widget-live-top10-memory-usage-20.04.0-5.el7.centos.noarch
centreon-web-20.04.3-5.el7.centos.noarch
centreon-broker-cbd-20.04.5-1.el7.centos.x86_64
centreon-trap-20.04.3-5.el7.centos.noarch
centreon-plugin-Applications-Monitoring-Centreon-Map4-Jmx-20200602-094050.el7.centos.noarch
centreon-plugin-Applications-Monitoring-Centreon-Database-20200602-094050.el7.centos.noarch
centreon-database-20.04.3-5.el7.centos.noarch
centreon-plugin-Applications-Monitoring-Centreon-Central-20200602-094050.el7.centos.noarch
centreon-poller-centreon-engine-20.04.3-5.el7.centos.noarch
centreon-engine-extcommands-20.04.2-1.el7.centos.x86_64
centreon-release-20.04-1.el7.centos.noarch
centreon-widget-hostgroup-monitoring-20.04.0-5.el7.centos.noarch
centreon-connector-20.04.0-2.el7.centos.x86_64
centreon-broker-20.04.5-1.el7.centos.x86_64
centreon-auto-discovery-server-20.04.2-2.el7.centos.noarch
centreon-plugin-Operatingsystems-Linux-Snmp-20200602-094050.el7.centos.noarch
centreon-plugin-Hardware-Ups-Standard-Rfc1628-Snmp-20200602-094050.el7.centos.noarch
centreon-plugin-Applications-Databases-Mysql-20200602-094050.el7.centos.noarch
centreon-base-config-centreon-engine-20.04.3-5.el7.centos.noarch
centreon-widget-httploader-20.04.0-5.el7.centos.noarch
centreon-widget-graph-monitoring-20.04.0-5.el7.centos.noarch
centreon-widget-live-top10-cpu-usage-20.04.0-5.el7.centos.noarch
centreon-connector-ssh-20.04.0-2.el7.centos.x86_64
centreon-pp-manager-20.04.1-1.el7.centos.noarch
centreon-license-manager-20.04.2-1.el7.centos.noarch
centreon-common-20.04.3-5.el7.centos.noarch
centreon-plugin-Network-Cisco-Standard-Snmp-20200602-094050.el7.centos.noarch
centreon-clib-20.04.0-7.el7.centos.x86_64
centreon-widget-servicegroup-monitoring-20.04.0-5.el7.centos.noarch
centreon-widget-global-health-20.04.0-5.el7.centos.noarch
centreon-widget-engine-status-20.04.1-1.el7.centos.noarch
centreon-broker-core-20.04.5-1.el7.centos.x86_64
centreon-widget-service-monitoring-20.04.2-1.el7.centos.noarch
centreon-plugin-Hardware-Printers-Generic-Snmp-20200602-094050.el7.centos.noarch
centreon-plugin-Applications-Protocol-Dns-20200602-094050.el7.centos.noarch
centreon-widget-host-monitoring-20.04.3-2.el7.centos.noarch
centreon-engine-20.04.2-1.el7.centos.x86_64
centreon-widget-tactical-overview-20.04.0-5.el7.centos.noarch
centreon-gorgone-20.04.2-1.el7.centos.noarch
```

***Operating System***

*CentOS Linux release 7.8.2003 (Core)*

***Browser used***

- [ X ] Google Chrome
- [ ] Firefox
- [ ] Internet Explorer IE11
- [ ] Safari

Version: --

***Additional environment details (AWS, VirtualBox, physical, etc.):***

AWS EC2

### Steps to Reproduce

Yesterday one of our host encountered a problem and we received a HOST DOWN notification followed few minutes after by a HOST UP notification. When I check the host vents log page, I can't see any information about this event, but I can see events in the /var/log/centreon-engine.log. 

1. Log Into Centreon
2. Go to monitoring page > Monitoring > Status Details > Hosts > select a host
3. Go to "view logs for host <SELECTED HOST>"
4. Check up/down events dans choose a period

### Describe the received result

UP/DOWN events are not visibles despite the fact that they are visibles in /var/log/centreon-engine.log

### Describe the expected result

UP/DOWN events should be visible

### Logs

here you can see the centreon-engine logs and the related dashboard screen :

**centreon-engine logs (*if needed*)**

![image](https://user-images.githubusercontent.com/22217453/85264889-b178a780-b471-11ea-98ab-72c5919b0eb5.png)

**View logs for host screen**
![image](https://user-images.githubusercontent.com/22217453/85265304-5d21f780-b472-11ea-88fa-e812b55a8065.png)

**Comment by cgagnaire** [comment]
Hi @anthoc,
This behaviour has been fixed with Centreon Web 20.04.4 (https://docs.centreon.com/current/en/releases/centreon-core.html#bug-fixes). You can update then your platform.

**Comment by anthoc** [comment]
@cgagnaire Thanks ! :)
